code/hof-parameter.kt

```diff
@@ -1,7 +1,8 @@
-fun transform(initial: String, f: (String) -> String) = f(initial)
+fun transform(initial: String, transformer: (String) -> String) = transformer(initial)
```

Comment:
Naming function with a single letter (`f`) is not the best practice. However, it was used to ensure the code would fit to a single view.